### PR TITLE
Fixes for building iOS / Android in parallel

### PR DESCRIPTION
**Pull request by LoungeKatt**

### CODE REVIEW

File: core/nullDC.cpp [hunk 1]
```diff
@@ -206,7 +206,11 @@ int dc_init(int argc,wchar* argv[])
 	
 	mem_map_default();
 
+#ifndef _ANDROID
```
Comment (skmp) [inline core/nullDC.cpp]: I don't quite get the point of this #ifdef

File: core/hw/maple/maple_cfg.h [hunk 2]
```diff
@@ -62,5 +62,9 @@ struct IMapleConfigMap
 	virtual ~IMapleConfigMap() {}
 };
 
+#ifndef _ANDROID
```
Comment (skmp) [inline core/hw/maple/maple_cfg.h]: I don't quite get the point of this #ifdef

File: core/nullDC.cpp (hunk 1 above)
Comment (LoungeKatt) [inline core/nullDC.cpp]: "Prep for Linux" which prevents that pull request from breaking Android.

File: core/hw/maple/maple_cfg.h (hunk 2 above)
Comment (LoungeKatt) [inline core/hw/maple/maple_cfg.h]: "Prep for Linux" which prevents that pull request from breaking Android.

**Pull request merged**